RetailSuiteAccounting/Operation/10_ManageReceipts.md

```diff
@@ -0,0 +1,194 @@
+
+The *Accounting* module offers three possible ways to manage and assign receipts to bookings:
+
+- Clicking the [ASSIGN RECEIPT] button in the *BOOKINGS* tab to assign a receipt to an already existing booking in the bookings list; see [Retrieve a receipt](#retrieve-a-receipt).
+- Clicking the *BOOK RECEIPTS* tab to assign a receipt available as a PDF and create a booking manually; see [Book a receipt](#book-a-receipt).
+- Clicking the *Supplier receipts* menu entry in the *EDIT* context menu to specifically assign a supplier receipt; see [Book a supplier receipt](#book-a-supplier-receipt)
+
+> [Info] *Supplier receipts* is a special feature that interacts with the *Purchasing* and *Payment processing* modules, and therefore the appropriate settings must be configured first.
+
+
+When a receipt is booked in the *BOOK RECEIPTS* tab, it is linked to the assigned booking. A ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon will be then displayed next to the booking in the bookings list. When clicking the ![Receipt](../../Assets/Icons/Beleg.png "[Receipt]") icon, the corresponding receipt is displayed.
+
+[comment]: <> (Work in progress!)
+
+
+## Retrieve a receipt
+
+The [ASSIGN RECEIPT] button in the *BOOKINGS* tab allows to retrieve and assign a receipt that has not been registered yet to an existing booking. Any unregistered receipts uploaded in the system are displayed in the *BOOK RECEIPTS* tab.  
+
+After being uploaded, the receipts are stored in the *New documents* folder in the *Documents* module. Once they are booked, they are moved automatically into the *Registered documents* folder and then placed in the appropriate subfolder according to the following structure:
+
+- 10000-69999 personal account numbers in case of a debtors'/customers' document
+- 70000-79999 personal account numbers in case of a creditors'/suppliers' document
+
+
+#### Prerequisites
+
+- The accounting is configured via the configuration wizard, see [Run the accounting wizard](../Integration/01_RunAccountingWizard.md).
+- A fiscal year is selected, see [Select fiscal year](./01_SelectFiscalYear.md).
+- A manual booking is created, see [Create a manual booking](./04_CreateManualBooking.md).
+- A receipt is uploaded in PDF format in the *Documents* module.
+
+#### Procedure
+
+*Accounting > Select fiscal year > Book > Tab BOOKINGS*
+
+![Assign receipt](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AssignReceipt.png "[Assign receipt]")
+
+1. Select the booking to be assigned a receipt from the bookings list in the *BOOKINGS* tab.  
+The existing booking details are displayed in the input line fields.
+
+2. Click the [ASSIGN RECEIPT] button.  
+The *Attach Document* window is displayed.
+
+  ![Attach document](../../Assets/Screenshots/RetailSuiteAccounting/Book/Bookings/AttachDocument.png "[Attach document]")
+
+3. Select the document to be assigned to the booking from the *New documents* folder.
```

Comment:
Lieber in zwei Schritten? 
1. Click the *New documents* folder in the side bar. -> All unassigned documents are displayed
2. Select the document to be assigned in the list of documents.
oder so ähnlich...